app/operations/mec_check/initiate_application_mec_checks.rb

```diff
@@ -53,9 +53,14 @@ def get_applicant_checks(application_payload, job)
       end
       Success(application_payload)
     rescue StandardError => e
+      add_errors({ job: job }, "Rescued while processing applicants error: #{e}", :get_applicant_checks)
```

Comment:
can we also add `hbx_id: application_payload["hbx_id"]` to the error message. This can be useful later.

Reply:
Yeah, I was thinking about it just now, thanks.